core/src/postprocessing/parallel_fiber_estimation/11_resample_fibers_in_file.tpp

```diff
@@ -267,8 +267,6 @@ void ParallelFiberEstimation<BasisFunctionType>::stretchMeshAtCornersInFile(
               sin(indexPhi)); // sin(phi) = g/h = y/maxR => maxR = y/sin(phi)
 
         double relativeR = fabs(indexR / maxR);
-        double m = 0.9; // 0.9        // parameter 0 << m <= 1, how much the
-                        // edge elements will be stretches, higher m means more
```

Comment:
Unrelated but intentional side change?

Reply:
Yes, I just saw a warning that m is not used and decided to change it here.